students/JasneetChandok/Lesson03/slicinglab.py

```diff
@@ -29,7 +29,7 @@ def elements_reversed(seq):
 
 def first_last_middle_thrid(seq):
 	'''Last third, then first third, then the middle third in the new order'''
-	new_sequence = seq[-3:][::-1] + seq[:3] + seq[(len(seq)//3):(len(seq)//3)*2]
+	new_sequence = seq[-3:] + seq[:3] + seq[(len(seq)//3):(len(seq)//3)*2]

```

Comment:
the first part will always be the first 3 and last 3 -- we want the first full third of the sequence!